packages/create-plugin/templates/datasource/provisioning/datasources/datasources.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+datasources:
+  - name: '{{ pluginName }}'
+  - type: '{{ normalize_id pluginName orgName 'datasource' }}'
+    access: proxy
+    isDefault: true
```

Comment:
Should use betteer `isDefault: false`?. We might find a bunch of plugins all trying to be default and generating start up errors if more people start adopting this template